atat_provisioning_wizard_api.yaml

```diff
@@ -1264,6 +1264,10 @@ x-amazon-apigateway-gateway-responses:
   # response handler will be used for responses we are not explicitly customizing.
   ACCESS_DENIED:
     statusCode: 403
+    responseParameters:
+      gatewayresponse.header.Access-Control-Allow-Headers: '''*'''
```

Comment:
We'll re-use whatever parameterization strategy we devise to set these properly right?